css/style.css

```diff
@@ -834,9 +848,22 @@ code {
 }
 
 .gd-on-translations .gp-content h2 {
-    font-size: 19px;
-    font-weight: 500;
-    line-height: 20px;
+	display: block!important;
+	width: 100%!important;
+    font-size: 19px!important;
+    font-weight: 500!important;
+    line-height: 20px!important;
+	padding: 10px 8px 9px!important;
+	border-radius: 5px!important;
+	background-color: rgb(0 115 170 / 5%)!important;
```

Comment:
How about `10%` :)

Reply:
I tried 10% and I found that it was like putting back a background color, whereas 5% corresponds to: ok we pass everything in white, but still we let imagine the area with a "cloud" :D